tests/test_future_py3.py

```diff
@@ -0,0 +1,23 @@
+from tests import has_async_await
+
+def _run_async(coroutine):
+    import asyncio
+    loop = asyncio.get_event_loop()
+    f = asyncio.ensure_future(coroutine, loop=loop)
+    return loop.run_until_complete(f)
+
+
+@has_async_await
+def test_future_supports_await_syntax(future):
+    async def _get_value():
+        return await future
```

Comment:
Please format with Black. I think it'll add a blank line after the inner function.